sale_triple_discount/tests/test_sale_triple_discount.py

```diff
@@ -112,13 +111,3 @@ def test_04_sale_order_triple_discount_invoicing(self):
         self.assertEqual(self.so_line2.discount3,
                          invoice.invoice_line_ids[1].discount3)
         self.assertEqual(self.order.amount_total, invoice.amount_total)
-
-    def test_05_sale_order_triple_discount_globally_rounding(self):
```

Comment:
Why are you removing the test? It's a valid use case that should be checked.

Reply:
Test added